mrp_bom_product_details/README.rst

```diff
@@ -0,0 +1,46 @@
+Bom product details
```

Comment:
Follow this file https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
please